setup.py

```diff
@@ -1,12 +1,25 @@
-from setuptools import setup, find_packages
+import versioneer
+from setuptools import find_packages, setup
+
 import archapp
 
+with open("requirements.txt", "rt") as fp:
```

Comment:
If we want this to build/install properly with pypi (do we want this?) then requirements.txt needs to be in the manifest as well so that setup.py can be run properly

Reply:
Got it 👍 

Yeah, we have it on pypi, so we may as well!